packages/utils/pack-up/src/node/core/pkg.ts

```diff
@@ -12,39 +12,67 @@ import { Logger } from './logger';
 
 import type { Export } from './exports';
 
+const record = (value: unknown) =>
+  yup
+    .object(
+      typeof value === 'object' && value
+        ? Object.entries(value).reduce<Record<string, yup.SchemaOf<string>>>((acc, [key]) => {
+            acc[key] = yup.string().required();
+
+            return acc;
+          }, {})
+        : {}
+    )
+    .optional();
+
 /**
  * The schema for the package.json that we expect,
  * currently pretty loose.
  */
 const packageJsonSchema = yup.object({
   name: yup.string().required(),
   version: yup.string().required(),
+  description: yup.string().optional(),
+  author: yup.lazy((value) => {
+    if (typeof value === 'object') {
+      return yup
+        .object({
+          name: yup.string().required(),
```

Comment:
Just checking, do we actually require name or should it be optional too?

Reply:
https://docs.npmjs.com/cli/v10/configuring-npm/package-json#people-fields-author-contributors afaik `name` is required 👍🏼